### PR TITLE
Adding nlp-primitives and nlp-primitives-complete

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by thehomebrewnerd** [comment]
@conda-forge-admin, please lint

**Comment by thehomebrewnerd** [comment]
@conda-forge-admin, please ping team

I am not clear on why the linter is failing as I have not deleted the example recipe. Any advice?

**Comment by synapticarbors** [comment]
@thehomebrewnerd -- this is an error with the linter and can be ignored. A fix should be released soon.

**Comment by thehomebrewnerd** [comment]
@synapticarbors Thank you! 

Also it seems the win_64 test failed, perhaps due to a lost connection. Should I just retrigger the CI run with the bot or could there be something else wrong that is causing this failure? 

Sorry, for the basic question, but this is my first contribution, so still trying to understand how everything works here.

**Comment by thehomebrewnerd** [comment]
@conda-forge-admin, please restart ci

**Comment by thehomebrewnerd** [comment]
@conda-forge-admin, please ping team

The windows CI test is failing with `OSError: [WinError 1455] The paging file is too small for this operation to complete`. Not exactly sure what is causing this, but it could be a resource issue as one of our tests requires a relatively large download to complete. Is is possible to execute this test with more resources available?

**Comment by thehomebrewnerd** [comment]
@conda-forge-admin, please lint

File: recipes/nlp_primitives/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  home: https://github.com/FeatureLabs/nlp_primitives
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: An open source python library with natural language processing primitives, intended for use with Featuretools.
+
+extra:
+  recipe-maintainers:
+    - thehomebrewnerd
+    - rwedge
```
Comment (scopatz) [inline recipes/nlp_primitives/meta.yaml:72]: CC @rwedge @thehomebrewnerd - are you willing to be maintainers?

Comment (thehomebrewnerd) [reply recipes/nlp_primitives/meta.yaml:72]: Yes!

Comment (rwedge) [reply recipes/nlp_primitives/meta.yaml:72]: Yes!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**